app/controllers/application_controller.rb

```diff
@@ -1,3 +1,29 @@
 class ApplicationController < ActionController::Base
   protect_from_forgery
+
+  before_action :require_login
```

Comment:
I was taking a queue from the rails guides on this one.  http://guides.rubyonrails.org/action_controller_overview.html#filters